make/env.mk

```diff
@@ -17,6 +17,8 @@ CGO_ENABLED := 1
 # Update the arch to arm64 but only for Macs running on Apple Silicon (M1)
 ifeq ($(shell uname -ms),Darwin arm64)
 	GOARCH := arm64
+else ifeq ($(shell uname -ms),Linux aarch64)
```

Comment:
On which system would we find such a configuration?

Reply:
This was the uname -ms of the node I built the apollo ci image on so this is what it is within the container